jni/utilities/debug.cpp

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Moved debug code out of <utillities/utils.h> as it was getting confusing when this class was included just for debugging purposes.